### ISSUE TITLE
home.def.js will only work for first feature

### ISSUE THREAD

**Issue by RefuX** [Kunstmaan/angular-super-seed#4]
```
   expect(element(by.repeater('feature in features')).getText()).to.eventually.contain(feature);
```

Will always get the text of the first feature.

So if you changed your home.feature test to be:
    _Then I should see the feature "Feature specific configuration"_

It would fail.

Which seems a little surprising.

**Comment by SpoBo** [comment]
You're right. I'd like this test to be more robust but haven't invested the time yet to get to know protractor. Do you have any idea what the expect statement should be like if we wanted to check all features in the repeater?
